Datasets/data-connect-dataset-calendarview.md

```diff
@@ -11,6 +11,15 @@ ms.custom: datasets:dataset-name
 
 The CalendarView_v0 dataset provides occurrences, exceptions, and single instances of events, based on the calendar view from users' calendars.
 
+NOTE:
+
+- Currently, the MGDC platform ONLY supports extracting valid users. Trying to extract invalid users will result in no data to be returned for such users. Valid users must have the following:
+    * A valid mailbox (i.e. users having mailbox that is **not in inactive state**, **not soft-deleted** or **not hosted on-Prem**)
+    * User account enabled (i.e. accountEnabled should not be set to false)
```

Comment:
Please rephrase "accountEnabled should not be set to false" to "accountEnabled should be true". Since this is the condition being checked while "accountEnabled should not be set to false" may indicate that absent value can also be accceptable

Reply:
As discussed. this is not needed